recipes/billingegroup-meta/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "billingegroup-meta" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  skip: True  # [py36 or win]
+  number: 0
+
+requirements:
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - seaborn
+    - fabio
+    - pims
+    - yaml
```

Comment:
I think you want `pyyaml` here. `yaml` is the c library. `pyyaml` is the python bindings that gives you the ability to `import yaml`.